File: Coders/CursoRegex/conjuntos/conjuntos.py
import re

texto = '1,2,3,4,5,6,a.b c!d?e[f'
texto2 = 'Jão não vai passear de moto'

print(re.findall(r'[0246]', texto))
print(re.findall(r'n[aã]o', texto2))

File: Coders/CursoRegex/conjuntos/conjuntos.js
const texto = '1,2,3,4,5,6a.b c!d?e[f'
const texto2 = 'Jão não vai passear na moto'

console.log(texto.match(/[02468]/g))
console.log(texto2.match(/n[aã]o/g))
